packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -573,6 +590,15 @@ export class Stack extends Construct implements ITaggable {
     };
   }
 
+  /**
+   * Returns the parent of a nested stack.
+   *
+   * @deprecated use `nestedStackParent`
+   */
+  public get parentStack() {
```

Comment:
I would be okay skipping this backwards compat thingy. It was `@experimental` for a reason...

Reply:
I'll leave it for now to avoid churn, since I think this API will change again soon.